### ISSUE TITLE
Make it an open source project instead of ending support

### ISSUE THREAD

**Issue by Foxy-null** [microsoft/WSA#560]
### Is your feature request related to a problem? Please describe

_No response \***from Microsoft\***_

### Describe the solution you'd like

Please I really need it for my daily tasks!😭

### Please specify the version of Windows Subsystem for Android

2311.40000.5.0

**Comment by miles5600** [comment]
yes @xhan65

**Comment by ghost** [comment]
yes, what a pitty

**Comment by MrSluffy** [comment]
plus 999999999

**Comment by XDRSZYHRL** [comment]
yes

**Comment by xbotuk** [comment]
yes please

**Comment by DemuraAIdev** [comment]
yes please

**Comment by MOONOTUS** [comment]
yes,please

**Comment by GitHub-Creeper** [comment]
yes, and also i heard someone on reddit says that closed-source software needs licensing to work, this is true, but missing, we can entirely switch closed-source projects to open-source and then choose a free license, and reduce cost and gain trust, why do they end support in such a good app?

**Comment by z505985755** [comment]
yes,please

**Comment by Mikrocontroler** [comment]
This would be an improve for everyone using WSA, and for Microsoft aswell, because Windows would be able to keep one of its best features (WSA)

**Comment by hohoaisan** [comment]
WSL is now open source. Hopefully WSA got the same fate

https://github.com/microsoft/WSL/issues/1
